floris/simulation/floris.py

```diff
@@ -225,48 +224,24 @@ def finalize(self):
     ## I/O
 
     @classmethod
-    def from_file(cls, input_file_path: str | Path, filetype: str = None) -> Floris:
-        """Creates a `Floris` instance from an input file. Must be filetype
-        JSON or YAML.
+    def from_file(cls, input_file_path: str | Path) -> Floris:
+        """Creates a `Floris` instance from an input file. Must be filetype YAML.
 
         Args:
             input_file_path (str): The relative or absolute file path and name to the
                 input file.
-            filetype (str): The type to export: [YAML | JSON]
 
         Returns:
             Floris: The class object instance.
         """
-        input_file_path = Path(input_file_path).resolve()
-        if filetype is None:
-            filetype = input_file_path.suffix.strip(".")
-
-        with open(input_file_path) as input_file:
-            if filetype.lower() in ("yml", "yaml"):
-                input_dict = load_yaml(input_file_path)
-            elif filetype.lower() == "json":
-                input_dict = json.load(input_file)
-
-                # TODO: This is a temporary hack to put the turbine definition into the farm.
-                # Long term, we need a strategy for handling this. The YAML file format supports
-                # pointers to other data, for example.
-                # input_dict["farm"]["turbine"] = input_dict["turbine"]
-                # input_dict.pop("turbine")
-            else:
-                raise ValueError("Supported import filetypes are JSON and YAML")
+        input_dict = load_yaml(Path(input_file_path).resolve())
         return Floris.from_dict(input_dict)
 
-    def to_file(self, output_file_path: str, filetype: str="YAML") -> None:
-        """Converts the `Floris` object to an input-ready JSON or YAML file at `output_file_path`.
+    def to_file(self, output_file_path: str) -> None:
+        """Converts the `Floris` object to an input-ready YAML file at `output_file_path`.
 
         Args:
             output_file_path (str): The full path and filename for where to save the file.
-            filetype (str): The type to export: [YAML | JSON]
         """
         with open(output_file_path, "w+") as f:
-            if filetype.lower() == "yaml":
-                yaml.dump(self.as_dict(), f, default_flow_style=False)
-            elif filetype.lower() == "json":
-                json.dump(self.as_dict(), f, indent=2, sort_keys=False)
-            else:
-                raise ValueError("Supported export filetypes are JSON and YAML")
+            yaml.dump(self.as_dict(), f, default_flow_style=False)
```

Comment:
I think we just need to update the keyword arguments to include `sort_keys=False`, to keep the same behavior. It's not in the original version, but I believe it should be, especially since the JSON had it, which was previously the more used form.

Reply:
Yep good catch

Reply:
Fixed. Thanks for looking through this